plugins/inputs/webhooks/plex/README.md

```diff
@@ -0,0 +1,45 @@
+# plex webhooks
+
+You should configure Plex's Webhooks to point at the `webhooks` service. To do this, follow the instructions in the [Plex webhooks support documentation](https://support.plex.tv/articles/115002267687-webhooks/). Webhooks are a premium feature and require an active Plex Pass Subscription for the Plex Media Server account.
+
+## Events
+
+The titles of the following sections are links to the full payloads and details for each event. The body contains what information from the event is persisted. The format is as follows:
+```
+# TAGS
+* 'tagKey' = `tagValue` type
+# FIELDS
+* 'fieldKey' = `fieldValue` type
+```
+The tag values and field values show the place on the incoming JSON object where the data is sourced from.
+
+**Tags:**
```

Comment:
I could imagine many of these tags being fields, right now it feels like virtually every event will create a new series.  Think about if the series are the server events (play, pause, resume, stop) or if the series is the history of a title.